pkgs/development/python-modules/oslo-utils/default.nix

```
@@ -67,12 +67,21 @@ buildPythonPackage rec {
     pyyaml
   ];
 
-  checkPhase = ''
+  disabledTests = [
+    # https://bugs.launchpad.net/oslo.utils/+bug/2054134
+    # netaddr default behaviour changed to be stricter
+    "oslo_utils.tests.test_netutils.NetworkUtilsTest.test_is_valid_ip"
+    "oslo_utils.tests.test_netutils.NetworkUtilsTest.test_is_valid_ipv4"
+  ];
+
+  checkPhase = let
+    disabled = lib.optionalString (lib.length disabledTests > 0) "-E '${lib.concatStringsSep "|" disabledTests}'";
```

```suggestion
    disabled = "-E '${lib.concatStringsSep "|" disabledTests}'";
```
I think we can just hardcode this